### PR TITLE
chore/switch-to-local-host

### PR DESCRIPTION
**Pull request by tav**

### CODE REVIEW

File: src/lib/config/index.ts [hunk 1]
```diff
@@ -44,11 +44,22 @@ export const LOGGING = set(
...
+export const POSTGRES = set(
+	process.env.POSTGRES || 'postgres://user:password@host:5433/lpg'
+)
+
 export const SESSION_SECRET = set(
 	process.env.SESSION_SECRET ||
 		'dcOVe-ZW3ul77l23GiQSNbTJtMRio87G2yUOUAk_otcbL3uywfyLMZ9NBmDMuuOt'
 )
 
+export const UI_SERVER = set('lpg.local.cshr.digital:3001', {
```
Comment (superafroman) [inline src/lib/config/index.ts]: These should be env variables configured on the host - we shouldn't need to change code to deploy to a new environment.

File: src/lib/config/index.ts [hunk 2]
```diff
@@ -44,11 +44,22 @@ export const LOGGING = set(
 	}
 )
 
+export const POSTGRES = set(
+	process.env.POSTGRES || 'postgres://user:password@host:5433/lpg'
```
Comment (superafroman) [inline src/lib/config/index.ts]: Guessing this isn't really meant to be part of this change, but obviously `@host` won't do much.

File: src/management-ui/page/courses/list.html [hunk 3]
```diff
@@ -6,22 +6,21 @@ <h1 class="heading-large">Courses</h1>
...
-                                    <!-- FIXME: not portable -->
-                                    <a href="http://lpg.dev.cshr.digital:3001/courses/{{course.uid}}/">Review</a>
-                                </td>
-                            </tr>
+                        <tr>
+                            <th scope="row">
+                                <a href="/courses/{{course.uid}}/edit">{{course.title}}</a>
+                            </th>
+                            <td>
+                                <a href="//{{config.UI_SERVER}}/courses/{{course.uid}}/">Review</a>
```
Comment (superafroman) [inline src/management-ui/page/courses/list.html]: This was a hack to let me easily check things. It won't make it into private beta and it won't work once we add auth to the management ui. If we actually want to allow preview we'd need to preview it through the management ui and block xapi events etc.. tldr: I don't think there's an actual need for configuring the UI host or necessarily exposing config to templates.

**Comment by tav** [comment]
[Add support for the new .local host and the new environments](https://trello.com/c/udn1FIQF/439-add-support-for-the-new-local-host-and-the-new-environments)

**Pull request merged**